index.rst

```diff
@@ -215,6 +215,15 @@ Welcome to PyTorch Tutorials
    :link: advanced/super_resolution_with_onnxruntime.html
    :tags: Production
 
+.. Code Transformations with FX
+.. customcarditem::
+   :header: Building a Convolution/Batch Norm fuser in FX
+   :card_description: Build a simple FX interpreter to record the runtime of op, module, and function calls and report statistics
```

Comment:
are the card description and images correct?  They look like they belong to other tutorials?

Reply:
Description is wrong, but not sure what to put for the image. @jamesr66a is there any reason you chose this image for the performance profiling for FX? https://github.com/pytorch/tutorials/pull/1319/files#diff-54a294a5d016e1a8e98bc95668ed84a99a9edd5c10394d9a2b1ee848006e98a7R223

Reply:
I put that because it's just a generic PyTorch logo: 
![image](https://user-images.githubusercontent.com/4685384/108142067-e3cabb00-7079-11eb-89aa-8466845ecb36.png)

And wasn't sure if I should copy that into a new filename or if we should come up with some logo to use